### DISCUSSION TITLE
Video Previews Fail When Using S3

### DISCUSSION THREAD

**Discussion by mjnewman** [pimcore/pimcore#8495] [Q&A / Support]
Video previews within the asset library are not working when using S3 for media storage.

We've tested two nearly identical Pimcore installations, one using S3, one using default storage. Video previews work when using default storage and fail when using S3.

Is this a known issue? And if so, are there any solutions to make this work?

Any help would be greatly appreciated!

**Comment by jdreesen** [comment]
Maybe #8335 and #8377 fix this (will be in the next patch release).

**Comment by ghost** [comment]
see https://github.com/pimcore/pimcore/issues/5007
